packages/graphback/src/components/Logger.ts

```diff
@@ -0,0 +1,23 @@
+
+
+/**
+ * Offers basic logging component providing ability to print some extra messages
+ */
+export interface GraphbackMessageLogger {
+    /**
+     * @param message message to log
+     * @param level additional context information to log
+     */
+    // tslint:disable-next-line: no-any
+    log(message: string, context?: any): void
```

Comment:
@wtrocki did you ever tried to use opentracing + jaeger? or elasticsearch? I see both tools as an implementation of the GraphbackMessageLogger but for both of them to work well in addition to the log message is best practice to provide also some key=value tags to filter the logs more easily. Another requirement for jaeger in order to trace requests across a microservice is to put a session id in the request header (or somewhere else) and to report the session id with the log, but it should be possible to retrieve the session id from the context.
Maybe I'm out of topic here but I wanted to say that maybe and additional arguments to log interface could be useful to provide these key=value

Reply:
Love this. Yes.. This is just the beggining to setup some places in the code for architecture reasons. This logging interface is wrong - it do not support levels etc. 
I just wanted to provide the whole architecture ion the code and start documenting it properly. Logging was the easiest one to have as it will give some message in the server that can be useful for offline use cases etc.